io.mpv.Mpv.appdata.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015-2019 gnome-mpv -->
+<component type="desktop">
+ <id>io.mpv.Mpv</id>
+ <metadata_license>CC-BY-SA-4.0</metadata_license>
+ <project_license>GPLv2+ and LGPLv2+</project_license>
+ <name>mpv</name>
+ <summary>a free, open source, and cross-platform media player</summary>
+ <launchable type="desktop-id">io.mpv.Mpv</launchable>
+ <translation type="gettext">mpv</translation>
+ <content_rating type="oars-1.1" />
+ <provides>
+  <id>io.mpv.Mpv.desktop</id>
+ </provides>
+ <description>
+  <p>
+   Mpv is a movie player based on MPlayer and mplayer2. It supports a wide variety
+of video file formats, audio and video codecs, and subtitle types. Special
+input URL types are available to read input from a variety of sources other
+than disk files. Depending on platform, a variety of different video and audio
+output methods are supported.
+  </p>
+  <p>Features:</p>
+  <ul>
+   <li>Drag and drop playlist</li>
+   <li>Loading external configuration files</li>
+  </ul>
+ </description>
+ <releases>
+  <release date="2019-10-02" version="0.29.1">
+   <description>
+    <p>
+     This release contains the following changes:
+    </p>
+    <ul>
+      <li>af_rubberband: Fix for A-V drift on seeking.</li>
+      <li>ao_alsa: Fixes to remove spam during pausing and end-of-file.</li>
+      <li>ao_jack: Fix for mpv attempting to connect to non-standard devices such as MIDI.</li>
+      <li>ao_openal: Drop support for Apple's bundled OpenAL due to lack of features to successfully build.</li>
+      <li>ao_pulse: Fix for audio buffer length calculation.</li>
+      <li>demux_lavf: Fix EDL playback of certain DASH streams.</li>
+      <li>demux_lavf: Mark v4l streams as not seekable.</li>
+      <li>demux_raw: Fix for gapless playback from raw audio input.</li>
+      <li>drm_atomic: Fix to enable video output with devices without a video overlay plane.</li>
+      <li>encode: Fix for crashes in case of failure to write to output.</li>
+      <li>encode: Fix for encoding when utilizing lavfi-complex.</li>
+      <li>Fix to strip HDR peak metadata if attached to SDR video (<a class="issue-link js-issue-link" data-error-text="Failed to load issue title" data-id="357030303" data-permission-text="Issue title is private" data-url="https://github.com/mpv-player/mpv/issues/6111" data-hovercard-type="issue" data-hovercard-url="/mpv-player/mpv/issues/6111/hovercard" href="https://github.com/mpv-player/mpv/issues/6111">#6111</a>)</li>
```

Comment:
Remove the a class links. They do not seem to be useful anyway ("Failed to load issue title").

Reply:
By the way, just a small typo: There is a missing dot at the end of the sentence. :-)